src/utilities.h

```diff
@@ -29,6 +29,12 @@ limitations under the License.
 
 namespace libcellml {
 
+/**
+ * Base URLs of specification and example sites from which the Issue::url() will be constructed.
+ */
+const std::string baseSpecificationUrl = "https://cellml-specification.readthedocs.io/en/latest/reference/formal_and_informative/";
+const std::string baseExampleUrl = "TODO";
```

Comment:
I would make those `static const`. In fact, looking at that header file, everything should `static const` (and for `interfaceTypeToString`, we don't need `std::string` to be `const`.

Reply:
Fixed in e8889490